Formula/fff.rb

```diff
@@ -0,0 +1,15 @@
+class Fff < Formula
+  desc "Fast file-manager (written in bash)"
+  homepage "https://github.com/dylanaraps/fff"
+  url "https://github.com/dylanaraps/fff/archive/1.2.tar.gz"
```

Comment:
Bump to [1.4](https://github.com/dylanaraps/fff/releases/tag/1.4) as pointed out by @dylanaraps.